app/src/main/res/layout/content_main.xml

```diff
@@ -34,5 +34,6 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:menu="@menu/activity_main_drawer" />
+        app:menu="@menu/activity_main_drawer"
+        app:itemBackground="@animator/menu_item_background_selector"/>
```

Comment:
Does this match iOS? or did you have another reason for adding this background rectangle?

![Screenshot_20240817_202714](https://github.com/user-attachments/assets/c9ffeb53-42ee-4a9f-8474-42cad5a9092a)

Reply:
The reason for using the background rectangle is that we already have it in Figma.

Reply:
The figma has many variations and I did not see this there, do you have the link to the design?

Reply:
https://www.figma.com/design/V5mIYvVRm7dAfZYesOWLBX/Street-Care-App-Design-(2023-updated)?node-id=5453-24924&t=mua9P7cJDj9RAeGc-1

This link has all the figma designs